fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResourceSwagger.java

```diff
@@ -1266,6 +1268,15 @@ private PutLoansLoanIdChanges() {}
         public PutLoansLoanIdChanges changes;
     }
 
+    @Schema(description = "PutLoansLoanIdFlagRequest")
```

Comment:
Its not used anywhere...

Reply:
PutLoansLoanIdRequest can be extended with fraud and be used

Reply:
removed

Reply:
nothing changed